### PR TITLE
splicectl/api - add `get urls` returning core and database urls

### PR DESCRIPTION
**Pull request by RileyMcCuen**
## Description

Added a `splicectl get urls` command that generates urls for core and database endpoints. This is done by listing ingress resources and building paths from their specs.

## Motivation and Context

This used to be done by a kurls script, but integrating it into splicectl is convenient and now it builds the urls based on the most up to date information, and not statically like the kurls script did.

See [DBAAS-4212](https://splicemachine.atlassian.net/browse/DBAAS-4212?atlOrigin=eyJpIjoiYzc1NjFjNWQ4NmQxNGE2N2I5Nzc4ZjBhYmY3MTZlOWEiLCJwIjoiaiJ9)

## How Has This Been Tested?

Manual testing was done on this functionality. No unit testing was done, as nearly all of the new functionality relies on the kubernetes api which is difficult to mock in a test environ.

## Checklist

If the pull request includes user-facing changes, extra documentation is required:

- [ ] If the change is user facing, please ensure you add info in one of the [Changelog Inclusions](#changelog-inclusions) sections.

## Changelog Inclusions

- base feature note
  - **BREAKING** note on base feature
  - Basically whatever formatting we have here, just plain-text
	%> command example
- next feature
  - note on next feature

### Additions

- Added `splicectl get urls` command to dynamically generate important urls and print them to the console.

### CODE REVIEW

**Comment by RileyMcCuen** [comment]
Example output from a test run against nonprod-gke-dev6:

```
Your urls are: 
Cloudmgr:                         https://nonprod-gke-dev6admin.gke.splicemachine-dev.io/
Cloudmgr:                         https://nonprod-gke-dev6.gke.splicemachine-dev.io/
Dev-Keycloak:                     https://nonprod-gke-dev6-keycloak.gke.splicemachine-dev.io/
Grafana:                          https://nonprod-gke-dev6-grafana.gke.splicemachine-dev.io/
Grafana:                          https://nonprod-gke-dev6-grafana-charts.gke.splicemachine-dev.io/
Kibana:                           https://nonprod-gke-dev6admin.gke.splicemachine-dev.io/kibana
Prometheus-Operator-Alertmanager: https://nonprod-gke-dev6-alertmgr.gke.splicemachine-dev.io/
Prometheus-Operator-Prometheus:   https://nonprod-gke-dev6-prom.gke.splicemachine-dev.io/
Splicectl-Api:                    https://nonprod-gke-dev6.gke.splicemachine-dev.io/splicectl
```

**Comment by splicemaahs** [comment]
This must have pulled in refactoring code that hasn't had a PR yet?

**Comment by RileyMcCuen** [comment]
> This must have pulled in refactoring code that hasn't had a PR yet?

Yeah, and it looks like I have yet to write that pr up yet. I will make sure to do that shortly.

**Comment by splicemaahs** [comment]
> Yeah, and it looks like I have yet to write that pr up yet. I will make sure to do that shortly.

Hopefully the conflicts will just disappear once we merge that one.

**Review by skolobov — APPROVED** [review]

**Review by splicemaahs — APPROVED** [review]
